src/features/contactmoment/ContactmomentNotitie.vue

```diff
@@ -0,0 +1,31 @@
+<template>
+  <div>
+    <utrecht-heading model-value :level="2">Notitieblok</utrecht-heading>
+    <textarea
+      ref="notitieRef"
```

Comment:
Visueel vind ik het niet heel duidelijk dat ik in het notitieblok kan klikken om notities te maken. Misschien met Gina bespreken?

Reply:
Dat was me niet opgevallen, maar dat scheelt idd

Reply:
done